### DISCUSSION TITLE
GSoC Projects Q/A

### DISCUSSION THREAD

**Discussion by Ankitsharma991** [qdrant/qdrant#1521] [General]
Hello all, Myself Ankit Sharma. I am a Second Year Computer Science student. I am familiar with Html, CSS, JavaScript, React, and Android(Kotlin). I am new to open source and want to contribute to Qdrant projects and learn many new things.
I am interested to contribute in Web UI for visualization and exploration. I found this project from Ideas for GSoC at Qdrant. Can anyone help me with how can I start my journey forward from here? @generall

**Comment by azayarni** [comment]
https://qdrant.to/gsoc23
